riscv/processor.cc

```diff
@@ -245,6 +245,17 @@ void processor_t::set_histogram(bool value)
 #endif
 }
 
+void processor_t::set_log_commits(bool value)
+{
+  log_commits_enabled = value;
+#ifndef RISCV_ENABLE_COMMITLOG
+  if (value) {
+    fprintf(stderr, "Commit logging support has not been properly enabled;");
+    fprintf(stderr, " please re-build the riscv-isa-sim project using \"configure --enable-commitlog\".\n");
```

Comment:
Done. I also realized that my branch was a few months old. I've merged up and tested. Instead of pushing back to this branch with -f, I've used github's merge option. Not sure I like it, but it does keep the PR comment history cleaner. I've never been a committer on a PR that's been merged up like this, so let me know if it causes any problems and I can certainly re-push with -f. Thanks again.